app_users/urls.py

```diff
@@ -1,8 +1,13 @@
-from django.urls import path
+from django.urls import path, include
+from django.contrib import admin
+from .views import SignUpView, ActivateAccount
 
 from . import views
 
 urlpatterns = [
     path('login_user/', views.login_user, name="login_user"),
-    path('logout_user',views.logout_user, name="logout_user"),
+    path('logout_user/',views.logout_user, name="logout_user"),
+   # path('register_user/', views.register_user, name="register_user"),
+    path('signup/', SignUpView.as_view(), name='signup'),
+    path('activate/<uidb64>/<token>/', ActivateAccount.as_view(), name='activate'),
```

Comment:
Although it's easier to search and match, it's recommended to use hyphens in URIs.